utils/typography.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
```

Comment:
In my opinion all of those components should be moved to their separate files in `components/shared`